### DISCUSSION TITLE
Supabase Libraries: Server function to validate JWT

### DISCUSSION THREAD

**Discussion by JoLoZ** [supabase/supabase#10725] [Feature Requests]
This suggestion is simple: Include a function for the server side of supabase.js & friends that validates a given JWT and returns `true`/`false` depending on whether it's valid and active.
This would result in one library less being used on average which is usually a good thing.

I hope it gets considered!
 -Jonas

**Comment by j4w8n** [comment]
You can use `client.auth.getUser('jwt')`. If it returns a user, then the jwt is valid and active.
